Calibration/EcalAlCaRecoProducers/python/ALCARECOEcalTrg_cff.py

```diff
@@ -1,4 +1,9 @@
 import FWCore.ParameterSet.Config as cms
+import HLTrigger.HLTfilters.htlHighLevel_cfi
 
-seqALCARECOEcalTrg = cms.Sequence()
+ecalTrgHLT = HLTrigger.HLTfilters.hltHighLevel_cfi.hltHighLevel.clone(
+#    eventSetupPathsKey='AlCa_EcalPhiSym*', # this is the HLT path that can be used                                                                                                          
+     eventSetupPathsKey='AlCaEcalTrg',
```

Comment:
The point is that we might want to "silence" it via trigger bits without affecting the rest, therefore I'd be in favour of having a separate key (that might include whatever trigger list you want).
I'll prepare an updated payload for the re-reco GT with the `AlCa_EcalPhiSym*` path in it in the IOV consumed by the relval jobs, for this to pass the bot tests. 
M.